pkg/sentry/kernel/shm/shm.go

```diff
@@ -396,14 +348,9 @@ type Shm struct {
 	// mu protects all fields below.
 	mu sync.Mutex `state:"nosave"`
 
-	// key is the public identifier for this segment.
-	key Key
+	// obj defines basic fields that should be included in all SysV IPC objects.
```

Comment:
nit: Comment provides no new information, consider deleting.

Reply:
Done. Also in semaphore.